packages/snaps-sdk/src/jsx/validation.ts

```diff
@@ -514,6 +514,28 @@ export const SectionStruct: Describe<SectionElement> = element('Section', {
   ),
 });
 
+export const NotificationChildrenStruct = children(
+  // eslint-disable-next-line @typescript-eslint/no-use-before-define
+  [lazy(() => NotificationComponentsStruct)],
+) as unknown as Struct<SnapsChildren<GenericSnapElement>, null>;
```

Comment:
Do we want to allow any element as notification child? I thought we agreed to start with a limited selection first, like box (outer wrapper), text, bold, italic, link.

Reply:
It's not any element, I included a few more: `copyable`, `address`, `divider`, `value`, `row`, `tooltip`, `icon, `image`. I don't think it hurts to allow for some more organization and polish to the content being returned, these are all presentational components.